src/navigation/CollapsingNavigationMenu.module.scss

```diff
@@ -0,0 +1,44 @@
+:local {
+  .collapsing-navigation-menu {
+    height: 100%;
+    overflow: auto;
+    padding: 10px;
```

Comment:
I'd go with `rem` units for the padding and margin in this file or use the `<Spacer />` component.